ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -236,7 +236,9 @@ function prepare_next_steps() {
       "${dir}/auth/kubeconfig" \
       "${dir}/auth/kubeadmin-password" \
       "${dir}/metadata.json"
-
+  if [[ "${CLUSTER_TYPE}" == "vsphere" ]] && [[ -f ${SHARED_DIR}/template.yaml.patch ]]; then
+      head -n 1000 "${dir}/.openshift_install.log" > "${SHARED_DIR}/.openshift_install.log"
```

Comment:
I feel like the 1000 lines is still a big size of data, maybe we can use `grep` command to capture the wanted lines.